hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/ContainerTestUtils.java

```diff
@@ -207,16 +207,21 @@ public static KeyValueContainer setUpTestContainer(
         UUID.randomUUID().toString(),
         UUID.randomUUID().toString());
     keyValueContainerData.setSchemaVersion(schemaVersion);
+    keyValueContainerData.setState(ContainerProtos
```

Comment:
Is this change needed in the tests?  I removed this and the `parseKVContainerData` addition to the test util without affecting the tests.  We can omit them if not needed for tests.